inc/BrainPadDisplay.h

```diff
@@ -0,0 +1,25 @@
+#ifndef BPDISPLAY_H
+#define BPDISPLAY_H
+
+#include "Image.h"
+#include "MbedI2C.h"
+#include "stdint.h"
+
+namespace codal {
+   class BrainPadDisplay {  
+  
+		uint8_t data[2];
+		int DeviceAddress = 0x78;
+		uint8_t byte;
```

Comment:
Is this used?

Reply:
Yes inside the BrainPadDisplay.cpp line 14

Reply:
No, since we changed it in BrainPadDisplay.cpp to uint8_t there rather than the variable byte, it's redundant.